.travis.yml

```diff
@@ -0,0 +1,21 @@
+script:
+  - pushd source
+  - make -j2
```

Comment:
On second thought, please do use `latexmk` for the test. The regeneration of the xrefs and grammar and plots is not a transient operation; and thus `make` is a tool for the author, not the user (despite what we say in the readme). If `make` were a user-facing tool, then we wouldn't have the grammar and xrefs checked in.

Note also that just `make` doesn't regenerate anything, since it only runs `rebuild` by default.

In summary, the semantics of `latexmk -pdf std` seem to be exactly what we want.